src/typeDefs/inputs.ts

```diff
@@ -339,6 +339,8 @@ export const inputs = gql`
     lastName: String
     maritalStatus: MaritalStatus
     phone: UserPhoneInput
+    applangcode: String
```

Comment:
`appLanguageCode`

Reply:
Hey, 

Regarding the variable name; that the frontend uses `applangcode`, so should I change it there also, otherwise it would give errors?

Reply:
Yes please do that @Devesh326

Reply:
@Devesh326 stop using abbreviations, it helps no one reading abbreviations to make sense of things, most of the time your code should be self documenting

in that light actually make it `applicationLanguageCode`

if frontend makes wrong assumptions about things, it doesn't mean it will be enforced on the backend design, if frontend wants to make changes it has to first open an issue/feature request